packages/lib/defaultEvents.ts

```diff
@@ -137,7 +132,8 @@ export const getDefaultEvent = (slug: string) => {
   const event = defaultEvents.find((obj) => {
     return obj.slug === slug;
   });
-  return event || min15Event;
+  const evt = event || min15Event;
+  return evt;
```

Comment:
Any reason for this change?

Reply:
Actually not in the final state of things, nice catch. I did this for debugging purposes and just forgot to revert back. Thanks for the callout! 🙏